backend/TikTakToe.Engines/Engines/random/random.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TikTakToe.Core.Boards;
+
+namespace TikTakToe.Engines.Engines.random
+{
+    public class random : IEngine
+    {
+        public Core.Enums.Engines Engine => Core.Enums.Engines.Random;
+
+        private int board;
+        private Stopwatch sw = new Stopwatch();
+        private Random randomPicker = new Random();
+
+        public double Bench(int depth)
+        {
+            sw.Restart();
+            sw.Start();
+            SetPos(0);
+            sw.Stop();
+            return sw.ElapsedMilliseconds;
+
+        }
+
+        public int MakeMove(int move)
+        {
+            board = Domove(board, move);
+            List<int> legalMoves = new List<int>();
+            for(int i = 1; i <= 9; i++)
+            {
+                if(Domove(board, i) != 0)
+                {
+                    legalMoves.Add(i);
+                }
+;
```

Comment:
lost ";"?

Reply:
don't know how this even build succefully :(